Makefile

```diff
@@ -18,7 +18,7 @@ BSL_AWS_PROFILE ?= default
 # BSL_AWS_PROFILE ?= migration-engineering
 
 # vsl secret
-CREDS_SECRET_REF ?= cloud-credentials
+CREDS_SECRET_REF ?= cloud-credentials-aws
```

Comment:
just to confirm VSL test is broken, undo change afterwards

Reply:
in case you need more contexts, VSL requires cred that can trigger snapshot of disks on the cluster.. in this case, prow CI. Cluster is provisioned in prow CI account, and therefor needs to use their secret to trigger snapshots of those volumes.

This VSL secret is linked to the prow ci aws account.. whereas the normal cred we use elsewhere is for our department's account which is only responsible for buckets, not cluster/disks/native-snapshots in prow CI.

Reply:
do not if I understand

here what I want to confirm is that we miss adding credentials struct to VSL, as we do for BSLs. Which means, DPA will not reconcile true, because it will try to use default secret

Reply:
Default secret is what is used currently in e2e for VSL. Custom secret name only applies to BSL.

Reply:
It would be helpful to read https://github.com/openshift/oadp-operator/blob/master/docs/developer/testing/%20MULTI_CLOUD_TESTING_UPDATES.md

Reply:
this was authored when VSL did not have the option to specify custom credential name. thus only default credentials were usable in VSL at the time.

Reply:
Hmm I did not know VSL had 3 defaults

But I still believe makes sense to remove option, as BSL do not have this flag in E2E